skglm/experimental/tests.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
for organization: put a test folder in experimental, and a test_sqrt.py file inside